backend/service/github/github.go

```diff
@@ -95,7 +95,10 @@ func (s *svc) CreateIssueComment(ctx context.Context, ref *RemoteRef, number int
 	com := &githubv3.IssueComment{
 		Body: strPtr(body),
 	}
-	_, _, err := s.rest.Issues.CreateComment(ctx, ref.RepoOwner, ref.RepoName, number, com)
+	_, response, err := s.rest.Issues.CreateComment(ctx, ref.RepoOwner, ref.RepoName, number, com)
+	if response != nil {
+		s.scope.SubScope("github_client").Gauge("rate_limit_remaining").Update(float64(response.Rate.Remaining))
```

Comment:
looks good but move to a method invocation e.g.
```go
emitRateLimit(response)
```

i'm also going to look to see if it's possible to just hook the http client so every method doesn't need to remember to call that